### DISCUSSION TITLE
How can I add a non-Quarto rendered file to "Other formats"?

### DISCUSSION THREAD

**Discussion by coatless** [quarto-dev/quarto-cli#9980] [Q&A]
### Description

With [Including Other formats](Including Other Formats), I'm wondering what is the best approach to adding a link that is not rendered by Quarto.  Is it just to write some JavaScript that automatically appends to the location? Or is there a way I can influence the submenu creation via Lua?

**Comment by mcanouil** [comment]
That's the "envelope" which is not part of the AST so you can't use Lua.

~~In Manuscript project, there is "other links" but I don't think it applies to other project types.~~

EDIT: see <https://quarto.org/docs/output-formats/html-basics.html#code-links-and-other-links>

**Reply by coatless** [reply]
@mcanouil bees knees. Didn't realize that was there. 

Is there also no way to use a Lua API to add values automatically there? :)

**Reply by mcanouil** [reply]
Since those are metadata, a LUA filter might work to fill these YAML keys.

Although, as the website/html envelope is done in TypeScript, I cannot guarantee the LUA filter applies soon enough in the workflow.

**Reply by coatless** [reply]
@mcanouil no worries; the read the doc link was plenty :) Though, I probably have a bug report to file w.r.t a custom ipynb format & running a filter.
